extensions/database/src/com/google/refine/extension/database/cmd/SavedConnectionCommand.java

```diff
@@ -126,25 +132,36 @@ private void writeSavedConnectionResponse(HttpServletResponse response, Database
             writer.writeArrayFieldStart(DatabaseUtils.SAVED_CONNECTION_KEY);
             
             writer.writeStartObject();
-            writer.writeStringField("connectionName", savedConnection.getConnectionName());
+            
+            String sanitizedConnName = Encode.forHtml(savedConnection.getConnectionName());
```

Comment:
Intuitively I would rather do HTML escaping in the frontend. The backend should just expose the database credentials in their true form, it is the frontend's job to render them to the user.